package-lock.json

```diff
@@ -16,7 +16,7 @@
         "buffer": "6.0.3",
         "json-duplicate-key-handle": "1.0.0",
         "keccak": "3.0.1",
-        "protobufjs": "6.10.2"
```

Comment:
[comment]

I'm not familiar with js. Did you update this package manually, or does it automatically update?

Reply:
I have updated it because it was an warning raised by `npm audit` which is automatically run when installing the dependencies